connector/src/lib.rs

```diff
@@ -28,6 +29,17 @@ impl<T, E: std::fmt::Debug> AnyhowWrap for Result<T, E> {
     }
 }
 
+#[derive(Clone, PartialEq, Debug)]
+pub enum FeedWrite {
+    Account(AccountWrite),
+    Snapshot(SnapshotWrite),
+}
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct SnapshotWrite {
+    pub accounts: Vec<AccountWrite>,
```

Comment:
I wonder if it's useful to have a separate SnapshotWrite when it's just a bundle of AccountWrites, consumers may be happy if they don't have to special case?